fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/entity/FabricTrackedDataRegistry.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.object.builder.v1.entity;
+
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.entity.data.TrackedDataHandler;
+import net.minecraft.entity.data.TrackedDataHandlerRegistry;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.impl.object.builder.FabricTrackedDataRegistryImpl;
+
+/**
+ * Allows registering custom {@link TrackedDataHandler}s in a reliable way.
+ */
+public final class FabricTrackedDataRegistry {
```

Comment:
Nit: Make this a `@ApiStatus.NonExtendable` interface.

Reply:
Why? All the methods are static, so it doesn't make sense for the class to be an itf.

Reply:
I thought this was part of the contributing guidelines, but its not. This is fine.